lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -316,17 +335,28 @@ class FontSize extends Gatherer {
     passContext.driver.on('CSS.styleSheetAdded', onStylesheetAdd);
 
     await Promise.all([
+      passContext.driver.sendCommand('DOMSnapshot.enable'),
       passContext.driver.sendCommand('DOM.enable'),
       passContext.driver.sendCommand('CSS.enable'),
     ]);
 
+    // We need to find all TextNodes that do not have legible text. DOMSnapshot.captureSnapshot is the
+    // fastest way to get the computed styles of every Node. Bonus, it allows for whitelisting properties.
+    // Once a bad TextNode is identified, its parent Node is needed. DOMSnapshot.captureSnapshot doesn't
+    // give the entire Node object, so DOM.getFlattenedDocument is used. The only connection between a snapshot
+    // Node and an actual Protocol Node is backendId, so that is used to join the two data structures.
```

Comment:
maybe just
`// Use DOMSnapshot.captureSnapshot to get the computed font-size style of every node.`
(the rest can be filled in below as it happens)